src/libraries/System.Linq.Expressions/src/System.Linq.Expressions.csproj

```diff
@@ -1,9 +1,10 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <FeatureInterpret>true</FeatureInterpret>
-    <TargetFrameworks>$(NetCoreAppCurrent)</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent);$(NetCoreAppCurrent)-iOS;$(NetCoreAppCurrent)-tvOS;$(NetCoreAppCurrent)-MacCatalyst</TargetFrameworks>
     <Nullable>enable</Nullable>
     <IsInterpreting>false</IsInterpreting>
+    <IsInterpreting Condition="'$(TargetsiOS)' == 'true' or '$(TargetstvOS)' == 'true' or '$(TargetsMacCatalyst)' == 'true'">true</IsInterpreting>
```

Comment:
What is the difference between IsInterpreting and FeatureInterpret?

Reply:
IIUIC  `IsInterpreting` is to adjust the use of `FEATURE_COMPILE` define, in case of `true` it says not to use compilation mode at all. `FEATURE_INTERPRET` define allows some additional mode "use a compilation or interpretation when it's possible" and it's used together with `FEATURE_COMPILE`.